config/samples/ccruntime-ssh-demo.yaml

```diff
@@ -52,7 +52,7 @@ spec:
     cleanupCmd: ["/opt/kata-artifacts/scripts/kata-deploy.sh", "reset"]
     runtimeClassNames: ["kata", "kata-clh", "kata-qemu"]
     postUninstall:
-      image: quay.io/confidential-containers/container-engine-for-cc-payload:4892b726eaf15e3932dd7990e196e92bd6f964d5
```

Comment:
On 0.1.0 we had a different runtime payload for the ssh-demo case. Have @stevenhorsman managed to unify the payloads?

Reply:
No, we'll still have a different payload for the ssh demo as part of `v0.2.0`.

Reply:
I think embedding the sample files into the main rootfs might not be the best idea as it will change the configuration, so we'd want more feedback before doing that. I've not against it, but when I proposed that before 0.1 the community seemed to want them separated.